### ISSUE TITLE
Large spaces in Live Preview vs. Reading Mode [BUG: Internal]

### ISSUE THREAD

**Issue by ryadaj** [tnichols217/obsidian-columns#24]
# Describe the bug
There are large spaces beneath headings and between list items in Live Preview vs. Reading Mode.

## Steps To Reproduce
Here is my snippet (I don't know how to do the nested ```` in GitHub)

```col-md
## :rif_draft: Field Notes

- Courses, CPD — **`Course about X with First Last on YYYY-MM-DD or in Term Year`**
- Entmoots — **`Entmoot about X or Project Name with First Last/Group on YYYY-MM-DD`**
- Events (physical, virtual) — **`Event Name on YYYY-MM-DD`**
- Feedback — **`Feedback on Project Name`**
````
```col-md
## :rif_leaf: In the Wild

- Books — **`Title by First Last`**
- Media— **`Title by First Last`**
- Poems — **`Title by First Last`**
- Quotes (good writing, idioms, proverbs) — **`summary`**
```
```col-md
## :fas_inbox: Inbox

- Fleeting notes
- Notes in progress
- Random
```

## Expected behavior
Normal spacing

## Screenshots
<img width="1512" alt="Screen Shot 2022-06-15 at 12 05 36" src="https://user-images.githubusercontent.com/82765301/173875087-66a6b4eb-bbb6-484f-b38d-bd99f523dd49.png">

<img width="1512" alt="Screen Shot 2022-06-15 at 12 05 33" src="https://user-images.githubusercontent.com/82765301/173875095-acd19bdb-202d-49c7-88b3-76152818e00f.png">

<img width="661" alt="Screen Shot 2022-06-15 at 12 11 52" src="https://user-images.githubusercontent.com/82765301/173875193-2bb2c4e9-8c9e-4dc8-83ed-75cdcb0dae59.png">

## System Info
OS: Mac OS X Monterey 12.4
Obsidian Version: Insider 0.15
Plugin Version: 1.1.7

**Comment by ryadaj** [comment]
This wasn't happening before I updated to 1.1.7

**Comment by tnichols217** [comment]
Fixed in commit [d065bf3](https://github.com/tnichols217/obsidian-columns/commit/d065bf32e89c2f09bcce94e8c80b67ff1e7b70ee) version [1.1.8](https://github.com/tnichols217/obsidian-columns/releases/tag/1.1.8)

By using 
```css
.columnParent {
    white-space: normal;
}

.cm-preview-code-block .admonition-content .columnParent p {
    white-space: pre-wrap;
}
```
snippet
